### DISCUSSION TITLE
INP Performance for Chakra-UI modals on low-end devices

### DISCUSSION THREAD

**Discussion by dz3n** [chakra-ui/chakra-ui#8731] [Q&A] [answered]
Is anyone experiencing issues with web vitals, specifically INP (Interaction to Next Paint)? I'm noticing that every Chakra UI Modal takes more than 200ms to load on **low-end mobile phones**, which is flagged as "needs improvement" in the Google Console. Is there anything I can do to improve this performance?

**Comment by segunadebayo** [comment, marked as answer]
v2 `Modal` is frozen, so we are not tuning INP there. v3 `Dialog` is the one to use going forward.
